src/app/components/BookList/BookList.jsx

```diff
@@ -17,21 +18,18 @@ const BookList = ({
       : null
   );
 
-  const { displayDate = {}, displayAge } = displayInfo;
-  const booksfound = `${picksCount} Book${picksCount === 1 ? '' : 's'} Found`;
-
   return (
     <div className="booklist-section nypl-column-three-quarters">
-      <h2 id="list-title" tabIndex="0">
-        {displayDate.month} {displayDate.year} Picks for {displayAge}
-        <span>{booksfound}</span>
-      </h2>
+      <div className="list-title-container">
+        <ListTitle displayInfo={displayInfo} picksCount={picksCount} idPrefix="" />
```

Comment:
You don't need `idPrefix=""` if the default value of that prop is an empty string.

Reply:
updated